src/NLog/LogFactory.cs

```diff
@@ -1165,7 +1165,7 @@ internal LogFactory LoadConfiguration(string configFile, bool optional)
 #if SILVERLIGHT
                     throw new System.IO.FileNotFoundException($"Failed to load NLog LoggingConfiguration from file {actualConfigFile}");
 #else
-                    var message = CreateFileNotFoundMessage(configFile);
+                    var message = CreateFileNotFoundMessage(_configLoader.GetDefaultCandidateConfigFilePaths(configFile));
```

Comment:
Not sure I understand. Why is `_configLoader.GetDefaultCandidateConfigFilePaths` no inside the try-catch-block?

Reply:
Well I prefer static methods, and `_configLoader.GetDefaultCandidateConfigFilePaths(configFile)` is lazy. But maybe it's a bit too much coincidence

Reply:
I was also doubting about this, so 2 hints = fix. :)

pushed!